doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-file-build.adoc

```diff
@@ -0,0 +1,112 @@
+[id="proc-kogito-microservices-deploying-on-ocp-console-file-build_{context}"]
+= Deploying {KOGITO} microservices on OpenShift using file build and OpenShift web console
+
+You can build and deploy your {KOGITO} microservices from a single file, such as a Decision Model and Notation (DMN), Drools Rule Language (DRL), or properties file, or from a directory with multiple files. You can specify a single file from your local file system path or specify a file directory from a local file system path only. When you upload the file or directory to an OpenShift cluster, a new Source-to-Image (S2I) build is automatically triggered.
+
+The operator uses the following custom resources to deploy domain-specific microservices (the microservices that you develop):
+
+* `KogitoBuild` generates an application from a file and produces a runtime image.
+* `KogitoRuntime` starts the runtime image and configures it as per your requirements.
+
+.Prerequisites
+* The Red Hat Process Automation Manager {KOGITO} Operator is installed.
+* The `oc` OpenShift CLI is installed and you are logged in to the relevant OpenShift cluster. For `oc` installation and login instructions, see the https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/cli_tools/openshift-cli-oc[OpenShift documentation].
+* You have access to the OpenShift web console with the necessary permissions to create and edit `KogitoBuild` and `KogitoRuntime`.
+
+.Procedure
+. Go to *Operators* -> *Installed Operators* and select *RHPAM Kogito Operator*.
+. To create the {KOGITO} build definition, on the operator page, select the *Kogito Build* tab and click *Create KogitoBuild*.
+. In the application window, use *Form View* or *YAML View* to configure the build definition.
++
+--
+At a minimum, define the application configurations shown in the following example YAML file:
+
+.Example YAML definition for a Quarkus application with {KOGITO} build
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rhpam.kiegroup.org/v1 # {KOGITO} API for this service
+kind: KogitoBuild  # Application type
+metadata:
+  name: example-quarkus  # Application name
+spec:
+  type: LocalSource
+----
+
+.Example YAML definition for a Spring Boot application with {KOGITO} build
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rhpam.kiegroup.org/v1 # {KOGITO} API for this service
+kind: KogitoBuild  # Application type
+metadata:
+  name: example-springboot  # Application name
+spec:
+  runtime: springboot
+  type: LocalSource
+----
+--
+
```

Comment:
We should add here also the NOTE about the Maven repository as it will use it when building

Reply:
Done, added the note.